api/Element.json

```diff
@@ -6893,9 +6894,15 @@
                 "prefix": "moz"
               }
             ],
-            "firefox_android": {
-              "version_added": true
-            },
+            "firefox_android": [
+              {
+                "version_added": "50"
+              },
+              {
+                "version_added": true,
+                "prefix": "moz"
```

Comment:
The data being mirrored is incorrect, this API has already been removed:
https://github.com/mozilla/gecko-dev/commit/6a9f5ff102c0f9b72dae14490608c22305a276ca

Can you add these APIs to the collector to find out when it exactly it was removed?

Reply:
Not sure why I mirrored this one when the original desktop data wasn't being updated.  I'll undo these changes and add the missing features for a follow-up PR later.

Reply:
Thanks!